prefs.js

```diff
@@ -2081,7 +2082,7 @@ const Preferences = class {
 
         // About Panel
 
-        this._builder.get_object('extension_version').set_label(Me.metadata.version.toString() + (Me.metadata.commit ? ' (' + Me.metadata.commit + ')' : ''));
+        // TODO this._builder.get_object('extension_version').set_label(this._settings.version.toString() + (this._settings.commit ? ' (' + this._settings.commit + ')' : ''));
```

Comment:
```suggestion
        const extension = ExtensionPreferences.lookupByURL(import.meta.url);
        const {metadata} = extension;
        this._builder.get_object('extension_version').set_label(metadata.version.toString() + (metadata.commit ? ' (' + metadata.commit + ')' : ''));
```

You can access the metadata like this